ts/features/newWallet/components/WalletCardBaseComponent.tsx

```diff
@@ -20,7 +20,7 @@ export const withWalletCardBaseComponent =
   >(
     CardContent: React.ComponentType<CardProps>
   ) =>
-  ({ cardProps, isStacked = false, testID }: ContentProps) =>
+  ({ cardProps, isStacked = true, testID }: ContentProps) =>
```

Comment:
Question: we are keeping this prop if the stacked behavior is removed everywhere else?

Reply:
In my opinion, the `isStacked` prop should be kept because not all cards have the `stacked` style - specifically, the last card in each group doesn't have this style. In `renderWalletCardFn`, we check which card is the last one and pass `stacked = false` for that card. Does this explanation make sense to you? Do you see any other approach we could take to handle this visual differentiation between cards?

Reply:
It makes sense, thank you